Externals/expr/include/expr.h

```diff
@@ -824,6 +846,16 @@ extern "C" {
         vec_push(&es, expr_const(num));
         paren_next = EXPR_PAREN_FORBIDDEN;
       }
+      else if (n > 1 && *tok == '"')
+      {
+        std::string string(tok, n - 1);
```

Comment:
Again, let's avoid introducing new C++ dependencies in a C library. Also, strdup() is a POSIX function and only standard C as of C23, so let's avoid that too. Let's do something like this instead:
```
      char* s = (char*)calloc(1, n - 1);
      if(s == NULL) {
        goto cleanup; /* allocation failed */
      }
      strncpy(s, tok + 1, n - 2);
```